### PR TITLE
Style fixes

### PR DESCRIPTION
**Pull request by krf**

### CODE REVIEW

File: ui/mainwindow.cpp [hunk 1]
```diff
@@ -115,31 +151,23 @@ MainWindow::MainWindow(QWidget *parent)
     , ui(new Ui::MainWindow)
     , m_stateManager(this)
 {
+
+    const auto styleOverride = gammarayStyleOverride();
+    if (styleOverride) {
+        applyStyle(styleOverride);
```
Comment (vkrause) [inline ui/mainwindow.cpp]: The applyStyle() hack is only needed for in-process mode, for the normal client qApp->setStyle() is a lot easier and more robust.

**Comment by vkrause** [comment]
+2

You can already test different styles by launching the client manually and using the -style option, but I'm fine with the env var solution as well.

File: ui/mainwindow.cpp (hunk 1 above)
Comment (krf) [inline ui/mainwindow.cpp]: Yeah, I wanted to keep this in line with what's done for the out-of-process mode. Plus the style changes are only local to the main window.

**Comment by krf** [comment]
Yep, I'm aware of that. It's way easier to be able to control this within an environment variable, though.  Working with `gammaray-client` is extra work.

**Pull request closed without merging**